bin/i18n/sync-out.rb

```diff
@@ -260,17 +260,13 @@ def distribute_translations(upload_manifests)
     next unless File.directory?(locale_dir)
 
     ### Dashboard
-    Dir.glob("i18n/locales/#{locale}/dashboard/*.{json,yml}") do |loc_file|
```

Comment:
I'd suggest we keep these changes in place, to smooth over any possible future efforts to switch back to json

Reply:
Sounds good. I'll pull in the fix from #35929 while I'm at it as well then.